src/PrestaShopBundle/Controller/Admin/Sell/Catalog/AttributeController.php

```diff
@@ -306,9 +357,15 @@ private function getErrorMessages()
             'Admin.Notifications.Error'
         );
 
+        $wrongValueMessage = $this->trans(
+            'A field is invalid.',
```

Comment:
Do we have an existing wording for when a field has a wrong value but we don't know what field it is ?